upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.12.yaml.template

```diff
@@ -219,8 +180,8 @@ spec:
           failureThreshold: 5
         readinessProbe:
           httpGet:
-            path: /health
-            port: 8080
+            path: /ready
+            port: 8181
```

Comment:
could that be a breaking change for some setup that use custom configuration? WDYT?

`ready` plugin is needed, that we add in our default config file ^^

Reply:
I cannot imagine this will be a problem.

Reply:
@olemarkus there were some users that use custom coreDNS config reporting this to be a breaking change for their setups. maybe good idea of adding a release note to guide of needed actions in such cases? WDYT?